### PR TITLE
Give felixfontein supershipit on crypto

### PR DESCRIPTION
**Pull request by gundalow**
##### SUMMARY
felixfontein has proved themselves with docker, so lets extend that to crypto where they've also done great work.

Also add in missing unit tests

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

**Comment by gundalow** [comment]
Need +1 from Tanner before merging

**Comment by abadger** [comment]
+1

**Review by mkrizek — APPROVED** [review]

**Comment by felixfontein** [comment]
Thanks everyone for your trust! :)

**Pull request merged**
